metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+# Share of Voice
+
+Can we communicate the value of our community as a comparison to something of value by our supporting business?
+
+## Description
+
+Ratio of conversation occurring by or about our community in comparison to some baseline. Also described as: Visibility, chatter, social media presence, top-of-funnel impact.
+
+## Objectives
+
+The metric supports the perception of continued value from community work in order to continue to secure funding. It highlights the value driven by community work by comparing it to something already valued by the organization. 
+
+## Implementation
+
+### Prerequisites
+
+Share of voice only works if you can agree to its implied assumptions ahead of time:
+
+- The more conversation about you, the better (which requires accepting the adage ‘there is no such thing as bad publicity’).
+- You have a public location worth measuring (public content like blogs or reviews, Twitter conversation, conference talks, StackOverflow tags).
+- There is a competitor that comes to mind, whether its internal competition where you compare community SOV to PR or AR efforts or external, where you focus on the top 3 companies in a given space (ex. comparing how N projects from X, Y, or Z companies compete for conversation around Data Science).
+- Your competitors are measurable in the same way you are, meaning you have a representative list of their activities similar to your own.
+- You have a trusted and repeatable system of measurement (ideally software used by multiple teams for multiple programs).
+
+If these statements are not applicable to your situation, abandon SOV immediately. It’s not for you at this moment. Rely on anecdotes or something else.
+
+### Visualizations
+
+One example made with PowerPoint:
+
+![sov-metric](https://user-images.githubusercontent.com/1744971/71270812-1b27c580-2318-11ea-9188-0a79ec700348.png)
```

Comment:
Please include the image as part of the PR so that it gets added to the repository.